.github/workflows/main.yaml

```diff
@@ -84,4 +84,43 @@ jobs:
           files: ./go_coverage.txt,./lcov_output.info
           name: ubuntu-${{ matrix.ubuntu }}-unittests
           fail_ci_if_error: true
-
+  windows:
+    name: Build with MSYS
+    runs-on: windows-2019
+    timeout-minutes: 90
```

Comment:
@matrixji I am curious about the reason for setting timeout-minutes for `90`, 30 minutes longer the run on ubuntu but without running ut. Does compile on windows  much slower than ubuntu?